src/main/java/foodwhere/ui/ReviewListPanel.java

```diff
@@ -30,6 +35,46 @@ public ReviewListPanel(ObservableList<Review> reviewList) {
         reviewListView.setCellFactory(listView -> new ReviewListViewCell());
     }
 
+    /**
+     * Handles the activity when the user clicks on the review in the list
+     *
+     * Adapted code from https://stackoverflow.com/questions/20635192/how-to-create-popup-menu
+     */
+    @FXML
+    public void handleMouseClicked() {
+        final Clipboard clipboard = Clipboard.getSystemClipboard();
+        final ClipboardContent content = new ClipboardContent();
+
+        final ContextMenu contextMenu = new ContextMenu();
+        MenuItem copyReviewName = new MenuItem("Copy Name");
+        MenuItem copyReviewDate = new MenuItem("Copy Date");
+        MenuItem copyReviewContent = new MenuItem("Copy Content");
+        MenuItem copyReviewTag = new MenuItem("Copy Tag");
+        contextMenu.getItems().addAll(copyReviewName, copyReviewDate, copyReviewContent, copyReviewTag);
+
+        reviewListView.setContextMenu(contextMenu);
+
+        copyReviewName.setOnAction((ActionEvent actionEvent) -> {
```

Comment:
Seems quite repetitive - perhaps we can consider abstracting this as a CopyMenuItem<Review> class?

Reply:
Thanks for the suggestion, I have abstracted out